src/main/asciidoc/predicates-attributes-handlers.asciidoc

```diff
@@ -241,11 +241,11 @@ The complete list of built in predicates is shown below:
 |Name           | Parameters  | Default Parameter | Additional context
 |auth-required	|	|	|
 |contains	|search: String[] (required), value: attribute (required)	|	|
-|directory	|value: attribute	|value	|
+|directory	|value: attribute	|value	| Only usable via undertow-handlers.conf
```

Comment:
This is only true in Wildfly. It would be more correct to say that it is only usable within the scope of a Servlet deployment (same for directory and dispatcher)